src/subsys/ngx_subsys_lua_util.c.tt2

```diff
@@ -828,9 +828,8 @@ ngx_[% subsys %]_lua_inject_ngx_api(lua_State *L, ngx_[% subsys %]_lua_main_conf
                 "if phase == 'init_worker' or phase == 'init' then\n"
                     "return\n"
                 "end\n"
-                "ngx_log(ngx_WARN, '[% subsys %] lua attempting to write to the "
-                         "lua global variable \\'', tostring(key), "
-                         "'\\'\\n', traceback())\n"
+                "ngx_log(ngx_WARN, 'writing to the global lua variable "
+                         "(key: \\'', tostring(key), '\\')\\n', traceback())\n"
```

Comment:
Maybe mention the reason here that "it may lead to race conditions among concurrent requests"?

Reply:
Also, maybe suggest the user use `local` to declare the variable at the same time?